aten/src/ATen/native/transformers/cuda/sdp_utils.h

```diff
@@ -622,7 +620,7 @@ inline bool use_mem_efficient_attention(sdp_params params, bool debug) {
       check_batch_size_and_num_heads,
       check_for_attn_mask,
       check_head_dim_size_mem_efficient,
-      check_head_dim_gt64_and_sm_ge86,
+      check_head_dim_gt64_and_sm_ge86_lt90,
```

Comment:
Did you also check that this bug is no longer showing up for mem-efficient kernel in sm90? I don't see that reflected in the tests

Reply:
??

Reply:
I ran https://github.com/Fuzzkatt/pytorch/blob/Fuzzkatt/enable_flash_attention_headdims_gt64_sm90/test/test_transformers.py#L1645 which should cover this case on H100 and it works, let me know if any additional testing is needed.